homeassistant/components/overkiz/executor.py

```diff
@@ -4,12 +4,22 @@
 from typing import Any
 from urllib.parse import urlparse
 
-from pyoverkiz.enums.command import OverkizCommand
+from pyoverkiz.enums import OverkizCommand, Protocol
 from pyoverkiz.models import Command, Device
 from pyoverkiz.types import StateType as OverkizStateType
 
 from .coordinator import OverkizDataUpdateCoordinator
 
+# Commands that don't support setting
+# the delay to another value
+COMMANDS_WITHOUT_DELAY = [
```

Comment:
This would be slightly more efficient if it was a `set()`

Reply:
Good suggestions @bdraco. I will have a look if we have other places where we don't need an ordered list and thus can use a set. Will do this in a follow up PR some day :).